69.md

```diff
@@ -0,0 +1,117 @@
+
+Pollers may specify multiple `p` keys, to allow participants to choose which recipient they zap, regardless of their vote choice. The option for such distribution of funds should also incentivize participation by increasing voter optionality, poller authenticity, and legitimacy of results by mitigating certain in-authentic attack vectors.
+
+By including a `value_maximum` limit, polls can stop single 'whale' votes discounting many smaller 'shrimp' votes. Likewise, by including a `value_minimum` limit, polls can make automated low-value vote flooding attacks prohibitively expensive. However, both limits remain optional to allow for freedom in poll design.
+
+The optional `consensus_threshold` is intended as a simple 'measuring bar', defined as the minimum percentage for any single `poll_option`'s percentage of the total tally to attain poll consensus status.
+
+A careful balancing of all poll attributes should enable pollers to conduct tailored polls that deliver meaningful and valuable outcomes.
+
+## Poll format
+
+A poll event:
+* MUST specify an `e` tag that includes a primary hosting relay URL
+* MUST specify 1 or more `p` tags, each including the same primary hosting relay specified in the `e` tag
+* MUST contain a primary description string, specified in the `content` field
+* MUST contain a `poll_options` tag, containing a serialized key-value array of vote options, formatted as below
+* MUST include at least 2 voting options, each with its own unique description string
+* SHOULD specify a `closed_at` time:
+  * when specified, MUST include a valid [`ots` tag](03.md), proving original poll publishing time
+  * a `closed_at` value of less than or equal to the `created_at` field indicates a poll SHOULD NOT be closed.
+* MAY specify a `value_maximum` satoshi value for zapped votes to be included in the tally
+* MAY specify a `value_minimum` satoshi value for zapped votes to be included in the tally
+  * `value_minimum` MUST be less than or equal to `value_maximum`, when both are specified
+* MAY include a `consensus_threshold` (0-100), representing a percentage threshold for any single vote option to achieve poll consensus
+  * a `consensus_threshold` of '0' indicates no threshold is specified.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded sha256 of the serialized event data>
+  "pubkey": <32-bytes lowercase hex-encoded public key of the event creator>,
+  "created_at": <unix timestamp in seconds>,
+  "kind": 6969,
+  "tags": [
+    ["e", <32-bytes hex of the id of the poll event>, <primary poll host relay URL>],
+    ["p", <32-bytes hex of the key>, <primary poll host relay URL>],
+    ["poll_options", "{
```

Comment:
I would change this to be one tag entry per poll option:

```
["poll_option","0","description"],
["poll_option","1","description"]
```

With that we avoid having to parse JSON again.

Reply:
@Cameri changed recursive json serialized `poll_options` tag to multiple poll_option tags for each vote option.
I also added a rule for limiting each participant to a single vote when min/max values are equal, while allowing participants to change their original vote choice.
Additionally, I added a Zap Voting section to clarify zap event submission rules for voting clients.